package.json

```diff
@@ -55,21 +57,22 @@
     "style-loader": "^0.13.1",
     "url-loader": "^0.5.7",
     "webpack": "^1.13.1",
+    "webpack-bundle-analyzer": "^2.8.2",
     "webpack-dev-server": "^1.14.1"
   },
   "dependencies": {
     "babel-runtime": "^6.9.2",
     "classnames": "^2.2.5",
-    "jquery": "^3.1.0",
     "lodash": "^4.14.1",
     "prop-types": "^15.5.7",
     "proxy-middleware": "^0.15.0",
     "react-datepicker": "^0.46.0",
     "react-input-autosize": "^1.1.0",
-    "react-select": "^0.9.1"
+    "react-select": "^1.0.0-rc.5"
```

Comment:
Oh shit, you're right. I think I tried upgrading this before and there are a lot of changes in component structure that require us to change most of our custom styles.